utils/modular_model_converter.py

```diff
@@ -256,7 +256,9 @@ def get_docstring_indent(docstring):
 def merge_docstrings(original_docstring, updated_docstring):
     # indent_level = get_docstring_indent(updated_docstring)
     original_level = get_docstring_indent(original_docstring)
-    if not re.findall(r"\n\s*Args:\n", updated_docstring):
+    if "REPLACES ORIGINAL DOCSTRING" in updated_docstring:
```

Comment:
IDK if this or just having `Args:` is better to just say it replaces!

Reply:
`Args:` is already [used in our doc builder](https://github.com/huggingface/doc-builder/blob/a3add1d7f9bde2450627f3a708c8a33061d19a21/README.md?plain=1#L464) as the keyword to start the arguments in the docs, so we would some other mechanism in this edge case.

I also think a magic string is ugly, but I couldn't think of a better error-proof pattern 😅 Happy to use any other pattern!